src/pymordemos/heat.py

```diff
@@ -59,11 +56,50 @@ def fom_properties(fom, w, stable=True, fig_poles=None, fig_bode=None):
         else:
             print('Skipped L_inf-norm calculation due to missing slycot.')
 
+    # Figure
+    if isinstance(fom, LTIModel) and fom.T is not None:
+        fig = plt.figure(figsize=(15, 8), constrained_layout=True)
+        fig.suptitle('Full-order model')
+        subfigs = fig.subfigures(1, 3)
+        subfigs1 = subfigs[1].subfigures(2, 1)
+        fig_bode = subfigs[0]
+        fig_poles = subfigs1[0]
+        fig_sv = subfigs1[1]
+        fig_time = subfigs[2]
+    elif not stable:
+        fig = plt.figure(figsize=(10, 4), constrained_layout=True)
+        fig.suptitle('Full-order model')
+        subfigs = fig.subfigures(1, 2)
+        fig_bode = subfigs[0]
+        fig_poles = subfigs[1]
+        fig_sv = None
+        fig_time = None
```

Comment:
Does the time-stepper assume stability? If not, we could also add it here, right?

Reply:
I guess not but could be dangerous. I'll try and see.